src/entry.tp

```diff
@@ -1,8 +1,10 @@
 {
     "sdk": 6,
-    "version": 1000,
+    "version": 1001,
```

Comment:
ahh forgot to revert this - but - you can change it to whatever you want when publishing the build